test/markets/markets.api-test.ts

```diff
@@ -17,46 +17,46 @@ describe('Markets', () => {
 
   it(`GET /markets`, async () => {
     const { status, body } = await api.get('/markets');
-    expect(status).toEqual(200);
+    expect(status).toBe(200);
```

Comment:
these changes are mainly driven by ESLint - it was disabled previously for test files